src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/dto/CreateSampleResponse.java

```diff
@@ -0,0 +1,7 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.dto;
```

Comment:
What is this being used for exactly? Since there are no methods for a sub class to extend we can probably get rid of it

Reply:
It's there because to allow you to return an instance of that either as `CreateSampleSuccessResponse` or `CreateSampleErrorResponse`. Since you must indicate what object a `ResponseEntity` is returning.  Do you have a better idea?